src/components/summary/_macro.spec.js

```diff
@@ -303,20 +303,23 @@ const EXAMPLE_SUMMARY_MULTIPLE_GROUPS = {
   ],
 };
 
+// Disable specific axe rules to accommodate the testing scenario
```

Comment:
I dont think this is explains why we are adding these rules because if we come back to this in some time we wont know why we made the change. We want something like "disable some axe definition list rules - solving a DAC issue (https://github.com/ONSdigital/design-system/pull/3053) caused these tests to fail. We would need a big refactor to the way summaries and errors work to get these to pass but these failing is not seen as something that is a big problem accessibility wise so we have decided to remove the testing of them here".

Reply:
updated